### PR TITLE
Latestversion compatibility

### PR DESCRIPTION
**Pull request by jeevitha90**
fix the Magento 2 plugin compatibility issue in 2.2 version

### CODE REVIEW

File: HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <head>
+        <script src="https://hps.github.io/token/2.1/securesubmit.js" src_type="url" />
+
+        <!-- Cert
+        <script src="https://hps.github.io/token/2.1/securesubmit.js" src_type="url" />
+        -->
+        <!-- PROD
+        <script src="https://api2.heartlandportico.com/SecureSubmit.v1/token/2.1/securesubmit.js" src_type="url" />
+        -->
+        <link src="HPS_Heartland/js/view/payment/securesubmit.js" id="HPS_SECURESUBMIT_IFRAME_CODE"/>
```
Comment (slogsdon) [inline HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml:22]: One script uses a `script` tag while the other uses a `link` tag. Is there a typo with one of these, or are both appropriate?

Comment (securesubmit-buildmaster) [reply HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml]: Yes shane, both can be apply to call js file. 
Ref link : http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/layouts/xml-manage.html

script

Comment (jeevitha90) [reply HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml:22]: Yes shane, both can be apply to call js file. 
Ref link : http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/layouts/xml-manage.html

script

Comment (slogsdon) [reply HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml:22]: Awesome! Thanks for the confirmation!

**Review by slogsdon — APPROVED** [review]

**Pull request merged**